_scripts/instruction-widget/templates/getting-started/windows.html

```diff
@@ -1,7 +1,112 @@
+<h1>Windows installation procedure</h1>
+
 <p>
-Certbot is currently only available for UNIX-like operating systems. Although
-EFF's Certbot might not work for your use case, there are many <a
-href="https://community.letsencrypt.org/t/list-of-client-implementations/2103">
+Certbot is now officially available for Windows. If you find that Certbot is not
+the most suitable Let's Encrypt client application for your use case, there are
+many <a href="https://community.letsencrypt.org/t/list-of-client-implementations/2103">
 other clients</a> written by other organizations and developers that you may be
 able to use to obtain a certificate from Let's Encrypt.
 </p>
+
+<h2>Important notes</h2>
+
+<p>
+This procedure follows the current Certbot implementation for Windows, in particular the fact that it installs as a system component, and requires administrative privileges. These instructions will be updated when a future version of Certbot <a href="https://github.com/certbot/certbot/issues/7883">switches to a different installation method</a>.
+No installers for HTTP servers are supported for now (Certbot for Windows can currently obtain your certificate from
+Let's Encrypt, but not install it into your web server application).
+
+<h2>Specific Windows system requirements and user knowledge requirements</h2>
+
+<ul>
+<li>The user needs to be familiar with the command-line interface (CLI), because Certbot is a pure CLI program.</li>
+<li>The user must use an account with administrative privileges to install and run Certbot.</li>
+<li>PowerShell and <code>CMD.EXE</code> are supported; both need to be started with elevated privileges before invoking Certbot.</li>
+<li>Path <code>C:\Certbot</code> must be writable by the current user.</li>
+</ul>
+
+<h2>Specific Windows limitations and configuration</h2>
+
+<ul>
+<li>All usual operations to create and manage an account, manage existing certificates, or select the ACME server, are supported.</li>
+<li>Only standalone, manual and webroot authenticator plugins are supported. DNS plugins will be available soon. This means that Certbot for Windows is currently unable to automatically renew wildcard certificates, since these require a DNS plugin in order to be renewed without user intervention.</li>
```

Comment:
One remark here: even if DNS plugins are not available, wildcard can still be issued if the user implements the proper hooks using the `manual` plugin to communicate with the DNS provider API, and create/clean TXT records on the DNS zone as required.

Reply:
While we do have a workaround to allow issuance of wildcard certificates I think it's still accurate to say that wildcard certificates are not automatically renewed.